match_FIPS_code.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
This script should be in it's own `scripts` directory, or eliminated all together where examples are provided in the README for certain use cases.

Reply:
I will put in a scripts directory and refer to them as examples in the README

Reply:
examples not yet put in the readme but need to talk aout how to intergrate them